libs/books/data-access/src/lib/+state/reading-list.reducer.spec.ts

```diff
@@ -32,24 +33,20 @@ describe('Books Reducer', () => {
       expect(result.ids.length).toEqual(3);
     });
 
-    it('failedAddToReadingList should undo book addition to the state', () => {
-      const action = ReadingListActions.failedAddToReadingList({
-        book: createBook('B')
-      });
+    it('failedAddToReadingList should return error for API failure on adding a book', () => {
```

Comment:
You can leave the test as is and update the reducer to pass the test instead.

The reason it is this way is for the UI to be updated optimistically, and any failures from API will undo the change.